src/automaticassessmentconfiguration.cs

```diff
@@ -3,8 +3,9 @@
 //
 // Authors:
 //	Alex Soto <alexsoto@microsoft.com>
+//	TJ Lambert <TJ.Lambert@microsoft.com>
 //
-// Copyright (c) Microsoft Corporation.
+// Copyright (2020) Microsoft Corporation.
```

Comment:
Do not change copyright headers, this is correct as it was

```suggestion
// Copyright (c) Microsoft Corporation.
```